libs/sdk-ui-tests-e2e/cypress/tools/widget.ts

```diff
@@ -120,6 +125,11 @@ export class Widget {
         return this;
     }
 
+    click() {
+        this.getElement().find(".is-selectable").click({ force: true });
```

Comment:
should click at the top left or top right to avoid click on the insight which has drill